package.json

```diff
@@ -1,42 +1,48 @@
 {
-  "name": "nemo-screenshot",
```

Comment:
Oh ok. I didn't see any of those probably because the entire package.json file is a diff due to the indent changes. That's a good motivation for separating meaty changes and cosmetic/stylistic ones into separate PRs. Might be a good thing to add to the "contribution" guide.